### ISSUE TITLE
Proposal: Connect LibreTime to Github Sponsors

### ISSUE THREAD

**Issue by Robbt** [LibreTime/libretime#1017]
Hey it has been suggested that we connect our OpenCollective and join the Github Sponsors program - See https://blog.opencollective.com/double-the-love/

To apply we will need at the minimum to decide whose e-mail is connected to the LibreTime org. Currently it is the email @hairmare used to create the LibreTime project. It might be nice for us to have a shared e-mail account that forwarded to different admins. But we can hash that out.

Anyways we could also just link the sponsor button to our opencollective in the meantime.

**Comment by paddatrapper** [comment]
Maybe team@libretime.org or admin@libretime.org? Not sure what features our current hosting has. It should have forwarding emails that can be used to forward emails from some @libretime.org email to relevant people. This could also be used for security issue submissions and webmaster emails from libretime.org

**Comment by Robbt** [comment]
I think libretime.org just uses github for hosting and so we would need to find a MX host.

**Comment by paddatrapper** [comment]
Sorry, I meant DNS provider. Often the DNS provider will have MX hosting tacked on. At least Gandi does and includes 1000 free forwarding addresses with any domain

**Comment by Robbt** [comment]
Yeah I think @hairmare has handled all of the DNS stuff and also registered the domain, nobody else had the login etc. If we wanted to figure out a secure way to share that we could or we could just defer to him for the time being. We do have enough funds in our open collective to handle this sort of minor expense at this point though.

**Comment by gusaus** [comment]
With regards to getting on the GitHub Sponsor waitlist (the first step in the process), we'll need to figure out the answers to the questions on the form (goto https://github.com/sponsors and click 'join the waitlist' and associate LibreTime to see the following - 

We’ll use the following questions to determine organization eligibility for GitHub Sponsors. Your answers don’t have to be long, but we’ll be looking for a thoughtful treatment of these key governance concerns in determining which teams are a good fit for this early version of GitHub Sponsors. If you’d like any guidance on how to answer these questions, feel free to contact us.

- What is the website for your organization?
- What are your organization's objectives and core values?
- How will your organization allocate the funds you receive from this sponsorship?
- How are decisions like "what to do with funding?" made in your organization? For example, do you have a single final decision maker, do you operate on consensus, do you take a vote, etc. Please detail your process and give a past example of that process in action.
- Are all of the key decision-makers onboard with the idea of this project or organization receiving funding through GitHub Sponsors? Please provide all of their names, emails, and GitHub handles.
- What developers are in your core development team but not a key decision-maker for whether or not the group receives funding? Please provide all of their names, emails, and GitHub handles.
- Collectively, how many hours per week does your core team work on open source projects associated with this organization, and on what kind of work do you spend that time? This can be a rough estimate.

Nonprofit status (don't worry, this is not a requirement to be in the program; we just need to know for tax purposes).

- [ ] No, this organization does not have nonprofit status
- [ ] Yes, this organization has nonprofit status and can receive tax-deductible donations from individuals (e.g., 501c3)
- [ ] Yes, this organization has nonprofit status but cannot receive tax-deductible donations from individuals (e.g., 501c6)

**Comment by gusaus** [comment]
Until that process is complete, it's pretty easy to create a sponsor button that links out to the existing collective - https://help.github.com/en/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository

@Robbt As discussed in chat, you probably have the GitHub creds to set that up.

**Comment by gusaus** [comment]
...and it looks like GitHub sponsors is coming out of beta https://github.blog/2020-05-12-github-sponsors-is-out-of-beta-for-sponsored-organizations/

> If your organization doesn’t have a bank account but you’re ready to receive funding, sign up with a fiscal host. We partnered with Open Collective to make it easy

Should we proceed?

**Comment by paddatrapper** [comment]
> Should we proceed?

I think so

**Comment by paddatrapper** [comment]
As for mailing stuff, I would be happy to set up a Gandi organisation that can manage the domain name if @hairmare doesn't want to manage it anymore or the registrar he is using can't handle multiple people managing a domain

**Comment by gusaus** [comment]
I'm not sure how these decisions are made - should a separate issue be created to sort out the email?

As mentioned in https://github.com/LibreTime/libretime/issues/1017#issuecomment-623698269, linking a sponsor button to Open Collective should be an easy first step.

**Comment by paddatrapper** [comment]
Or @hairmare can respond here

**Comment by gusaus** [comment]
> it's pretty easy to create a sponsor button that links out to the existing collective - https://help.github.com/en/github/administering-a-repository/displaying-a-sponsor-button-in-your-repository
> 
> @Robbt As discussed in chat, you probably have the GitHub creds to set that up.

What's the decision-making process for creating the sponsor button? Would 👍 from the active maintainers suffice?

**Comment by paddatrapper** [comment]
See PR #1048

**Comment by paddatrapper** [comment]
> What is the website for your organization?

http://libretime.org

> What are your organization's objectives and core values?

Open access to broadcasting for all

> How will your organization allocate the funds you receive from this sponsorship?

In line with our development goals and priorities?

> How are decisions like "what to do with funding?" made in your organization? For example, do you have a single final decision maker, do you operate on consensus, do you take a vote, etc. Please detail your process and give a past example of that process in action.

Open issues with a vote from the Core Maintainers? Where confidentiality is required, a private discussion and vote from the Core Maintainers?

> Are all of the key decision-makers onboard with the idea of this project or organization receiving funding through GitHub Sponsors? Please provide all of their names, emails, and GitHub handles.

@paddatrapper (Kyle Robbertze) - github@paddatrapper.com

> What developers are in your core development team but not a key decision-maker for whether or not the group receives funding? Please provide all of their names, emails, and GitHub handles.

I don't know what this question wants?

> Collectively, how many hours per week does your core team work on open source projects associated with this organization, and on what kind of work do you spend that time? This can be a rough estimate.

1-5 hours depending on the requirements. Issue triage, occasional code, support

**Comment by Robbt** [comment]
This is a good start. I do think we should figure out a process for determining what to use funds on before we start receiving funds. I'm not sure a vote by core maintainers is the best way to distribute non-trivial amounts of funds because the active core maintainers has been relatively small and we don't have a real process for adding new maintainers. Like maybe we should have semi-regular meetings to do the actual votes and/or strive for consensus before we start voting. Things have been going pretty smoothly so now is probably a good time to figure this stuff out.l

**Comment by paddatrapper** [comment]
A semi-regular meeting sounds like a good idea. Can do it on chat.libretime.org and publish logs somewhere open too

**Comment by gusaus** [comment]
> Like maybe we should have semi-regular meetings to do the actual votes and/or strive for consensus before we start voting.

Maybe we could pick up on the discussions started in #sustain - looks like we had at least on semi-regular meeting almost a year ago https://chat.libretime.org/libretime/pl/ku1pri47s7btjkpr3cmq77cp9e

With regards to this particular task, I don't think it would be a bad idea to come up with answers and make the submission to get on the Github sponsors waiting list.

**Comment by paddatrapper** [comment]
Maybe a monthly meeting then?

**Comment by gusaus** [comment]
> we should figure out a process for determining what to use funds on before we start receiving funds.

I think we can pick up from this comment https://github.com/LibreTime/libretime/issues/796#issuecomment-483845707 and subsequent discussion in https://chat.libretime.org/libretime/channels/sustain 

Possibly create a separate issue to work this out?
